src/render.ts

```diff
@@ -7,10 +7,17 @@ const canUseNewReactRootApi =
   reactDomVersion &&
   (reactDomVersion.startsWith('18') || reactDomVersion.startsWith('0.0.0'));
 
-export const renderElement = (node, outlet) => {
+declare let __webpack_public_path__: string | undefined;
```

Comment:
I don't think `declare` is needed here.
```suggestion
let __webpack_public_path__: string | undefined;
```

Reply:
removed